### DISCUSSION TITLE
PROJECTS need to be `organisation-wide` and `repository-wide`

### DISCUSSION THREAD

**Discussion by ttodua** [community/community#42147] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

It will be useful if projects will not be just `organisation` level, but instead, we need to create projects that are not (must not be) visible/existent in organization level projects, but instead, those projects are only for one `repository`.
so, you need to make two project management pages, organization-scope  VS repository-scope: 

![2022-12-19 22 42 06](https://user-images.githubusercontent.com/7117978/208497496-1efccddf-6edf-4d09-97ec-346a6acd7530.png)

**Comment by arg** [comment]
Totally agree, IMO the absence of repository-level projects is the biggest disadvantage over the classic (legacy) projects.
